r/tests/testthat/test-dplyr-string-functions.R

```diff
@@ -724,3 +724,67 @@ test_that("errors in strptime", {
     'Time zone argument not supported by Arrow'
   )
 })
+
+test_that("str_like", {
+  
+  df <- tibble(x = c("Foo and bar", "baz and qux and quux"))
+  
+  # This will give an error until a new version of stringr with str_like has been released
```

Comment:
It's probably best to `skip()` this instead of wrapping it in `expect_error()`, that way when the new stringr is released, it won't cause this test to start failing. Could you move it to after the other expectations just below, and add a `skip()` before it?

Reply:
That makes sense, done.